terraform/services/rds-api/variables.tf

```diff
@@ -0,0 +1,17 @@
+variable "app" {
+  description = "The application name (ab2d, bcda, dpc)"
+  type        = string
+  validation {
+    condition     = contains(["ab2d", "bcda", "dpc"], var.app)
+    error_message = "Valid value for app is ab2d, bcda, or dpc."
+  }
+}
+
+variable "env" {
+  description = "The application environment (dev, test, prod, sbx)"
+  type        = string
+  validation {
+    condition     = contains(["dev", "test", "prod", "sbx"], var.env)
+    error_message = "Valid value for env is dev, test, or prod."
```

Comment:
Missing sbx in the error message. Also, the expected order would be dev, test, sbx, prod